webserver/__init__.py

```diff
@@ -81,10 +81,6 @@ def create_app(debug=None):
     else:
         raise Exception('One or more redis cache configuration options are missing from config.py')
 
-    # Extensions
-    from flask_uuid import FlaskUUID
```

Comment:
this wasn't needed because it was already done in the `use_flask_uuid` argument to `CustomFlask`